90-facilitator-laptop/lab/content/labguides/ex-3.1/index.md

```diff
@@ -5,26 +5,26 @@ title: Managing Configurations
 
 #### Cluster configurations
 
-Now that we have several edge clusters (maybe?) running and (definitely) selectable under the same `student=#` label (one as the VM we're installing on, and maybe one as the pre-installed bare metal we're adopting), let's look at managing those clusters as part of an overall hybrid cloud infrastructure.
+Now that we have several edge clusters (maybe?) running and selectable under the same `student=#` label (one being the VM we're installing on, and one bare metal if available), let's look at managing those clusters as part of an overall hybrid cloud infrastructure.
 
-Normally, our selectors would be a bit more broad in order to manage a large group of clusters per region, use-case, or whatever's appropriate for what your customer is trying to accomplish. Today, we're just trying to make sure that everyone gets to see how ACM works on these edge clusters - so your selectors in the following exercises will be pretty narrowly scoped to just your student number, selecting one or two clusters.
+Normally, our selectors would be a bit more broad in order to manage a large group of clusters; per region, specific to a use case, or whatever's appropriate for what your customer is trying to accomplish. Today, we're just trying to make sure that everyone gets to see how ACM works on these edge clusters - so your selectors in the following exercises will be pretty narrowly scoped to just your student number, selecting one or two clusters.
```

Comment:
I agree, it is a little jarring. I added the semi-colon more because this sentence is listing reasons why we might use selectors. I'll try and rework it, to make it more smooth